js/index.js

```diff
@@ -40,3 +40,75 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+
+let logoImg = document.getElementById('cta-img');
+logoImg.src="./img/header-img.png";
+
+
+let middleImg = document.getElementById('middle-img');
+middleImg.src="./img/mid-page-accent.jpg";
+
+
+let navi = document.querySelectorAll('a');
+navi[0].textContent= "Services";
+navi[1].textContent= "Product";
+navi[2].textContent= "Vision";
+navi[3].textContent= "Features";
+navi[4].textContent= "About";
+navi[5].textContent= "Contact";
+                                                                                              
+navi.forEach(function(currentValue) {
+  currentValue.style.color='green';
+})
+
+
+let newPage = document.createElement('a');
+newPage.textContent="Account";
+
+let navigation = document.querySelector('nav');
+navigation.append(newPage);
+
+let newPageTwo = document.createElement('a');
+newPageTwo.textContent="Settings";
+
+navigation.prepend(newPageTwo);
+
+const topContent= document.getElementsByTagName('h1');
+topContent[0].textContent= "DOM is Awesome"; 
```

Comment:
good job getting the [0] index. 
to add linebreak, try `innerHTML` instead of `textContent`